ProcessLib/GroundwaterFlow/GroundwaterFlowProcess.h

```diff
@@ -14,7 +14,46 @@
 #include "ProcessLib/Process.h"
 #include "GroundwaterFlowFEM.h"
 #include "GroundwaterFlowProcessData.h"
+#include "ProcessLib/BalanceProcess/BalanceProcess.h"
 
+// TODO used for output, if output classes are ready this has to be changed
+#include "MeshLib/IO/writeMeshToFile.h"
+
+// TODO Copied from VectorMatrixAssembler. Could be provided by the DOF table.
+inline NumLib::LocalToGlobalIndexMap::RowColumnIndices
+getRowColumnIndices_(std::size_t const id,
```

Comment:
I think by now this method is already defined in some `*Util*.{h,cpp}` file. Same for the function below.